.github/workflows/ci-nix.yml

```diff
@@ -102,11 +102,32 @@ jobs:
 
       - name: Tests
         if: (github.ref == 'refs/heads/master' || matrix.build-in-pr) && matrix.run-tests
-        run: nix build -L .#ci.cli-test.all
+        run: |
+          nix build -L .#ci.cli-test.all --keep-failed
 
       - name: Wasm Tests
         if: (github.ref == 'refs/heads/master' || matrix.build-in-pr) && matrix.run-tests
-        run: nix build -L .#ci.wasm-tests
+        run: nix build -L .#ci.wasm-tests --keep-failed
+
+      - name: Prepare failed test build dirs
+        if: always()
```

Comment:
Does github really need this if?

Reply:
Otherwise if previous steps fail (tests), these actions won't execute, no?

Reply:
:thinking: 
Not sure, but whatever